src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/rca/hotheap/HighHeapUsageYoungGenRca.java

```diff
@@ -53,6 +55,9 @@
 
   private static final Logger LOG = LogManager.getLogger(HighHeapUsageYoungGenRca.class);
   private static final int PROMOTION_RATE_SLIDING_WINDOW_IN_MINS = 10;
+  private static final int ONE_HOUR_IN_MIN = 60;
+  private static final double GARBAGE_PROMOTION_THRESHOLD = .80;
+  private static final double FOLLOWER_CHECK_TIMEOUT_MS = 30d * 1_000;
```

Comment:
why is this called `FOLLOWER_CHECK_*` ? is this related to leader/follower checks in es?

Reply:
The idea is that this is related to leader/follower checks. I noticed that in a lot of Sev2s, when this issue causes node drops it's because GC time starts approaching this value.

This may need to be dynamic eventually, but for now a value around the default for our clusters should be good enough